.github/workflows/integration-test-object-suite.yaml

```
@@ -0,0 +1,64 @@
+name: Integration test CephObjectSuite
+on:
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  TestCephObjectSuite:
+    if: ${{ github.event_name == 'pull_request' && github.ref != 'refs/heads/master' && !contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-versions : ['v1.22.0']
```

We are running most of the suites on two versions for PRs, really to help see if something is intermittent.
```suggestion
        kubernetes-versions : ['v1.16.15', 'v1.22.0']
```

---

Good call. I'll do that for the integration tests on release also.